### ISSUE TITLE
Cannot whitelist KeepassXC for auto-type in MacOS 10.14 Mojave

### ISSUE THREAD

**Issue by ChristopherSchultz** [keepassxreboot/keepassxc#4607]
After upgrading to 2.5.4, auto-type (via a global hot-key) no longer works but instead shows an "Open System Preferences" dialog and gives me this message:

"KeePassXC requires the Accessibility and Screen Recorder permission in order to perform global Auto-Type. Screen Recording is necessary to use the window title to find entries. If you already granted permission, you may have to restart KeePassXC."

It doesn't appear that "Screen Recording" is a permission that I can grant in Mojave *at all*.

(I have another machine running Catalina where that preference *does* exist, but I can't figure out how to grant permission to KeePassXC... it doesn't have the "+/-" buttons that e.g. the "Accessibility" permission seems to have.)

Downgrading to 2.5.3 allows me to use auto-type again.

Any ideas?

**Comment by droidmonkey** [comment]
Restart your computer with 2.5.4 installed.

**Comment by ChristopherSchultz** [comment]
Duplicate of which issue?

**Comment by phoerious** [comment]
Of the one that is pinned to the very top of the issue list.

**Comment by ChristopherSchultz** [comment]
Aha, that issue's description is so long it looks like it's (merely) about mouse clicking on 10.15. It's issue #4573.

**Comment by droidmonkey** [comment]
Fixed the title, good point
